plugins/modules/zos_job_submit.py

```diff
@@ -801,9 +803,7 @@ def run_module():
         except IndexError:
             pass
         except Exception as e:
-            result["err_detail"] = "{1} {2}.\n".format(
-                "Error during job submission.  The output is:", job_output_txt or " "
-            )
+            result["err_detail"] = "Error during job submission."
```

Comment:
It looks like you corrected a blank message, given there is little we can provide the user they don't already know eg, job id, maybe we just be a bit more verbose such that it reads: "An error has occurred while submitting the requested job."

Reply:
Yes.  If job_output_txt is None, this line caused exceptions.  If the error is there, stderr is already populated.  I'll change the text.